airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -49,12 +49,18 @@ public PostgresDestination() {
   public JsonNode toJdbcConfig(JsonNode config) {
     final String schema = Optional.ofNullable(config.get("schema")).map(JsonNode::asText).orElse("public");
 
+    final StringBuilder jdbcUrl = new StringBuilder(String.format("jdbc:postgresql://%s:%s/%s",
+        config.get("host").asText(),
+        config.get("port").asText(),
+        config.get("database").asText()));
+
+    if (config.get("ssl").asBoolean()) {
+      jdbcUrl.append("?ssl=true&sslmode=require");
```

Comment:
I don't think we should have the `?` in here. Can we put it in the original url instead? It will make adding additional parameters harder in the future.

Maybe one way to do it is to have an array of options and we just join on `&` at the end and append it to the jdbcUrl